fiona/ogrext.pyx

```diff
@@ -225,7 +559,37 @@ cdef class FeatureBuilder:
     argument is not destroyed.
     """
 
-    cdef build(self, void *feature, encoding='utf-8', bbox=False, driver=None, ignore_fields=None, ignore_geometry=False):
+    cdef object driver
+    cdef object property_getter_cache
```

Comment:
I've added some state to the class so that we can reuse field getter and setters.